frontend/src/scenes/insights/ActionFilter/entityFilterLogic.ts

```diff
@@ -222,6 +231,10 @@ export const entityFilterLogic = kea<entityFilterLogicType<BareEntity, EntityFil
                     ...props.addFilterDefaultOptions,
                 },
             ])
+            // When adding a 2nd or later series, switch away from map view, as it only works for one series at a time
```

Comment:
Should this be a visual affordance? 

Disabling the button to add a second series and a message explaining why...

Rather than the user clicking a button and their insight selection changing out from under them

Reply:
Good point, I agree the insight changing could be too disruptive.

Instead, now there's no "Add graph series" button in map view (`entitiesLimit={1}`) and there's a tooltip in display type selection that quickly explains what "Map" view is.

Here it is in an insight with two series:

<img width="402" alt="Screen Shot 2022-04-05 at 14 45 42" src="https://user-images.githubusercontent.com/4550621/161756580-efcf8456-2252-4e34-ae6b-c4a5a3c12787.png">